packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -173,23 +175,31 @@ export class SignUp extends BaseResource implements SignUpResource {
     return this.attemptWeb3WalletVerification({ signature });
   };
 
-  public authenticateWithMetamask = async (): Promise<SignUpResource> => {
+  public authenticateWithMetamask = async (params: {
+    unsafeMetadata: SignUpUnsafeMetadata;
```

Comment:
🙃 Not a huge deal and simply for consistency's sake, we should introduce an `AuthenticateWithMetamaskParams` type similar to the `authenticateWithWeb3` method